nvm.sh

```diff
@@ -3286,7 +3286,7 @@ nvm() {
         FLAVOR="$(nvm_node_prefix)"
       fi
 
-      local EXIT_CODE
+      local EXIT_CODE=0
```

Comment:
this would break ksh, here's the proper way to initialize a variable:
```suggestion
      local EXIT_CODE
      EXIT_CODE=0
```